exercise_09/getDistanceMessageFromSumTo100.js

```diff
@@ -19,3 +19,25 @@
  * random (generated in function): 40
  * output: "Sum with value 70 isd left in 30 from number 100"
  */
+
+
+/**
+ * #9 :: Generates a random number between 0 and 100, adds it to the given number
+ * and returns a message if the sum exceeds 100 or not.
+ * 
+ * @param {number} value - The number to add the random number to.
+ * @returns {string} - The message indicating whether or not the sum exceeds 100.
+ */
+const getDistanceMessageFromSumTo100 = function (value) {
+    const randomNumber = Math.floor(Math.random() * 101);
+    const sum = value + randomNumber;
+  
+    if (sum > 100) {
```

Comment:
Avoid magic numbers. Use descriptive constants
```suggestion
    if (sum > THRESHOLD) {
```